cores/utils.py

```diff
@@ -0,0 +1,26 @@
+import jwt
+
+from django.conf import settings
+from ..orders.models import User
+
+from django.http import JsonResponse
+
+def logindeco(func):
+    def wrapper(self, request, *args, **kwargs):
```

Comment:
`*args, **kwargs`를 argument로 받는 이유가 무엇인가?

```suggestion
    def wrapper(self, request, *args, **kwargs):
```

Reply:
확장성: 토큰이 아닌 다른 값이 들어올 경우도 생각해서 추가하였습니다.